src/triage/component/catwalk/__init__.py

```diff
@@ -71,17 +89,17 @@ def generate_task_batches(self, splits, grid_config, model_comment=None):
     def order_and_batch_tasks(self, tasks):
         batches = (
             TaskBatch(
-                parallelizable=True,
+                key=BatchKey.QUICKTRAIN,
                 tasks=[],
                 description="Baselines or simple classifiers (e.g. DecisionTree, SLR)"
             ),
             TaskBatch(
-                parallelizable=False,
+                key=BatchKey.BIGTRAIN,
                 tasks=[],
-                description="Heavyweight classifiers with n_jobs set to -1."
+                description="Heavyweight classifiers."
             ),
             TaskBatch(
-                parallelizable=True,
+                key=BatchKey.MAYBETRAIN,
                 tasks=[],
                 description="All classifiers not found in one of the other batches (e.g. gradient boosting)."
```

Comment:
I think gradient boosting is now found in the `BIGTRAIN` batch, right? Maybe remove the example here?